src/storybook/stories/orders/OrderCustomerAddressesEditDialog.tsx

```diff
@@ -2,41 +2,49 @@ import Decorator from "@saleor/storybook/Decorator";
 import { storiesOf } from "@storybook/react";
 import React from "react";
 
-import { countries, order as orderFixture } from "../../fixtures";
 import OrderCustomerAddressesEditDialog, {
+  AddressEditDialogVariant,
   OrderCustomerAddressesEditDialogProps
-} from "./OrderCustomerAddressesEditDialog";
+} from "../../../orders/components/OrderCustomerAddressesEditDialog";
+import { countries, order as orderFixture } from "../../../orders/fixtures";
 
 const order = orderFixture("");
 
 const props: OrderCustomerAddressesEditDialogProps = {
   confirmButtonState: "default",
+  variant: AddressEditDialogVariant.CHANGE_CUSTOMER,
   loading: false,
   onClose: () => undefined,
   onConfirm: () => undefined,
   open: true,
-  errors: undefined
+  errors: undefined,
+  countries
 };
 
 storiesOf("Orders / OrderCustomerAddressesEditDialog", module)
```

Comment:
We can name it in some meaningful way, like "Change order customer" or whatever this component does